### PR TITLE
Update GOVERNANCE.md

### PR DESCRIPTION
**Pull request by stianst**
Instead of sending an email to the dev mailing list it will be better to have a PR against the maintainers list directly. This allows voting directly on the PR, and gives more visibility to the community.

### CODE REVIEW

**Review by ssilvert — DISMISSED** [review]
It makes perfect sense to use a PR to start the nomination process.

But I do want to put out a word of caution.  A discussion over who is allowed to be a maintainer is by nature something that is very personal.  If said discussion is put out in the open it can lead to serious issues within the organization.  I'd hate to see us in a situation where half the team approves a nomination and half does not.  It would be even worse if a contentious nomination is out on GitHub for all to see.

This is definitely outside my area of expertise so I don't want to pass judgement on how we should actually handle it.  This sort of problem is endemic in any organization that has a board of directors or in this case, a list of maintainers.  In the open source world, there have been plenty of fights over who is allowed to be a committer.

So my only advice is that the current maintainers take this under advisement.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -62,8 +62,7 @@ To become a maintainer, you need to demonstrate the following:
 * Ability to collaborate with the team
 * Helping the community
 
-A new maintainer must be proposed by sending an email to the [developer mailing list](https://groups.google.com/forum/#!forum/keycloak-dev).
-The email should include evidence of the above list.
+A new maintainer must be nominated by sending a pull request to the [Maintainers list]https://github.com/keycloak/keycloak/blob/master/MAINTAINERS.md. The PR description must include evidence to support the nomination.
```
Comment (hmlnarik) [inline GOVERNANCE.md:65]: Should this PR be sent by one of the existing maintainers?

**Comment by pedroigor** [comment]
Very good points from @ssilvert. Perhaps we should also define at least some basic requirements (or make them more explicit) for nomination.

The text `include evidence to support the nomination` is quite subjective. Perhaps we should explain better what are those pieces of evidence like:

*  Number and weight of contributions in one or more areas
*  Level of support and commitment of contributions made (mainly those that involve a specific component and/or SPI)
*  Discussions on mailing lists
*  Community help and feedback

I expect that some basic requirements should help us deciding and make the process easier and less subjective. And perhaps avoid issues with unexpected nominations.

**Comment by stianst** [comment]
With Stan's concerns and the fact that we know how a new maintainers mailing list, let's just use that as a place to propose new maintainers.

**Review by abstractj — APPROVED** [review]
LGTM based on the latest changes.

**Review by pedroigor — APPROVED** [review]

**Pull request merged**